### PR TITLE
[panw] Add session.start_time for ML jobs

### PR DESCRIPTION
**Pull request by aleksmaus**
## Proposed commit message

Add session.start_time for ML jobs

Addresses SDH: https://github.com/elastic/sdh-beats/issues/4367

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates https://github.com/elastic/sdh-beats/issues/4367

### CODE REVIEW

File: packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/traffic.yml [hunk 1]
```diff
@@ -319,7 +319,11 @@ processors:
         - 'yyyy/MM/dd HH:mm:ss'
         - 'strict_date_optional_time_nanos'
       on_failure: [{'append': {'field': 'error.message', 'value': '{{{ _ingest.on_failure_message }}}'}}]
-
+  - set:
+      if: ctx.panw?.panos?.parent_session?.start_time != null
+      field: session.start_time
+      value: '{{{panw.panos.parent_session.start_time}}}'
+
```
Comment (taylor-swanson) [inline packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/traffic.yml:326]: I'm not sure why this pipeline sets `panw.panos.parent_session.start_time` as well, perhaps it's leftover from a refactor or something. Shouldn't anything, though. I'd rather address any issues with duplicated processors in a separate issue (and more so if it were actually causing problems).

File: packages/panw/docs/README.md [hunk 2]
```diff
@@ -654,6 +654,7 @@ An example event for `panos` looks as following:
 | server.port | Port of the server. | long |
 | server.user.name | Short name or login of the user. | keyword |
 | server.user.name.text | Multi-field of `server.user.name`. | match_only_text |
+| session.start_time |  | date |
```
Comment (taylor-swanson) [inline packages/panw/docs/README.md:657]: This is my fault, should've mentioned that you should copy the description from the original panos field.

File: packages/panw/data_stream/panos/fields/fields.yml [hunk 3]
```diff
@@ -1043,3 +1043,5 @@
       type: boolean
     - name: x_forwarded_for
       type: boolean
+- name: session.start_time
+  type: date
```
Comment (taylor-swanson) [inline packages/panw/data_stream/panos/fields/fields.yml:1047]:
```suggestion
  type: date
  description: The start time of the session.
```

File: packages/panw/docs/README.md (hunk 2 above)
Comment (aleksmaus) [reply packages/panw/docs/README.md]: The original was pretty basic ```Time of session start.```. Updated.

File: packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/traffic.yml (hunk 1 above)
Comment (aleksmaus) [reply packages/panw/data_stream/panos/elasticsearch/ingest_pipeline/traffic.yml]: Removed this change from PR

**Review by taylor-swanson — APPROVED** [review]
LGTM

**Comment by aleksmaus** [comment]
buildkite test this

**Pull request merged**